src/plugins/discover/public/application/view_components/canvas/canvas.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { AppMountParameters } from '../../../../../../core/public';
+import { useOpenSearchDashboards } from '../../../../../opensearch_dashboards_react/public';
+import { DiscoverServices } from '../../../build_services';
+import { TopNav } from './top_nav';
+import { updateState, useDispatch, useSelector } from '../../utils/state_management';
+
+interface CanvasProps {
+  opts: {
+    setHeaderActionMenu: AppMountParameters['setHeaderActionMenu'];
+  };
+}
+
+export const Canvas = ({ opts }: CanvasProps) => {
+  const { services } = useOpenSearchDashboards<DiscoverServices>();
+  const interval = useSelector((state) => state.discover.interval);
+  const dispatch = useDispatch();
+
+  return (
+    <div>
```

Comment:
nit - should be just `<>` or else an OUI component - but is this just placeholder demo content?

Reply:
Yes this is placeholder content for now. Anan is adding the actual content inhere soon